services/venice-router/src/main/java/com/linkedin/venice/router/stats/AggRouterHttpRequestStats.java

```diff
@@ -29,14 +32,18 @@ public AggRouterHttpRequestStats(
       ReadOnlyStoreRepository metadataRepository,
       boolean isUnregisterMetricForDeletedStoreEnabled) {
     super(metricsRepository, metadataRepository, isUnregisterMetricForDeletedStoreEnabled);
+    // Disable store level non-streaming multi get stats reporting because it's no longer used in clients. We still
+    // report to the total stats for visibility of potential old clients.
+    isStoreStatsEnabled = !RequestType.MULTI_GET.equals(requestType);
```

Comment:
Should we just remove MULTI_GET? Do we have any use cases for this?

Reply:
Let's remove it when we actually remove the code to support it too.

Reply:
LGTM